### PR TITLE
feat: add round_ to jax.numpy

### LINKED ISSUE #20967: round_
related to https://github.com/unifyai/ivy/issues/8426

### PR DESCRIPTION
**Pull request by OzMoti**
close #20967

### CODE REVIEW

File: ivy/functional/frontends/jax/numpy/mathematical_functions.py [hunk 1]
```diff
@@ -549,6 +549,9 @@ def around(a, decimals=0, out=None):
 def round(a, decimals=0, out=None):
     return ivy.round(a, decimals=decimals, out=out)
 
+@to_ivy_arrays_and_back
+def round_(a, decimals=0, out=None):
+    return ivy.round(a, decimals=decimals, out=out)
```
Comment (kurshakuz) [inline ivy/functional/frontends/jax/numpy/mathematical_functions.py:554]: Hey! I have tested the implementation locally and it seems to be failing for the `AssertionError`. Can you please resolve it?
```
E       AssertionError:  the results from backend paddle and ground truth framework jax do not match
E        32768!=48896 
```
```
E       AssertionError:  the results from backend tensorflow and ground truth framework jax do not match
E        -inf!=-1.0 
```
```
E       AssertionError:  the results from backend numpy and ground truth framework jax do not match
E        -inf!=-1.0 
```

**Review by kurshakuz — CHANGES_REQUESTED** [review]

**Comment by OzMoti** [comment]
Would you please give me or give me a clue on how to solve this issue ? I couldn't find the solution.

**Comment by NiteshK84** [comment]
Hello @OzMoti, I apologize for the lack of response from the Ivy team regarding your PR. Would you mind resolving the merge conflicts first? Afterward, we can address any implementation failures that may have occurred.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**